util/intensity_explorer.py

```diff
@@ -392,10 +392,10 @@ def data_from_unmerged_mtz(filename):
     rtable["id"] = flex.int(rtable.size(), 0)
 
     # Now generate a corresponding experiment list.
-    id = flex.vec3_double([(1, 0, 0), (0, 1, 0), (0, 0, 1)])
+    _id = flex.vec3_double([(1, 0, 0), (0, 1, 0), (0, 0, 1)])
```

Comment:
This appears to be using miller index as an id - so `indices` might be better? or even just `ids`...